### PR TITLE
[TTS] replace pitch normalization params with ???

### PR DESCRIPTION
**Pull request by XuesongYang**
Signed-off-by: Xuesong Yang <1646669+XuesongYang@users.noreply.github.com>

# What does this PR do ?

Follow up the discussion in https://github.com/NVIDIA/NeMo/pull/5383, we decided to replace pitch norm params with ??? for all YAML configs just in case users forget to update with dataset-specific values.

**Collection**: [Note which collection this PR will affect]

# Changelog 
- Add specific line by line info of high level changes in this PR.

# Usage
* You can potentially add a usage example below

```python
# Add a code snippet demonstrating how to use this 
```

# Before your PR is "Ready for review"
**Pre checks**:
- [ ] Make sure you read and followed [Contributor guidelines](https://github.com/NVIDIA/NeMo/blob/main/CONTRIBUTING.md)
- [ ] Did you write any new necessary tests?
- [ ] Did you add or update any necessary documentation?
- [ ] Does the PR affect components that are optional to install? (Ex: Numba, Pynini, Apex etc)
  - [ ] Reviewer: Does the PR have correct import guards for all optional libraries?
  
**PR Type**:
- [ ] New Feature
- [ ] Bugfix
- [ ] Documentation

If you haven't finished some of the above items you can still open "Draft" PR.

## Who can review?

Anyone in the community is free to review the PR once the checks have passed. 
[Contributor guidelines](https://github.com/NVIDIA/NeMo/blob/main/CONTRIBUTING.md) contains specific people who can review PRs to various areas.

# Additional Information
* Related to # (issue)

### CODE REVIEW

File: examples/tts/conf/es/fastpitch_align_44100_ipa.yaml [hunk 1]
```diff
@@ -13,9 +13,10 @@ sup_data_types: ["align_prior_matrix", "pitch"]
 pitch_fmin: 65.40639132514966
 pitch_fmax: 2093.004522404789
 
-# Stats (per frame), train (these values depend on pitch_fmin and pitch_fmax)
-pitch_mean: 212.35873413085938
-pitch_std: 68.52806091308594
+# these frame-wise values depend on pitch_fmin and pitch_fmax, you can get values
+# by running `scripts/dataset_processing/tts/extract_sup_data.py`
+pitch_mean: ???  # e.g. 212.35873413085938
+pitch_std:  ???  # e.g.  68.52806091308594
```
Comment (XuesongYang) [inline examples/tts/conf/es/fastpitch_align_44100_ipa.yaml:19]: @rlangman please let me know what dataset these values were computed on. Then I will update here. Thanks.

**Review by redoctopus — COMMENTED** [review]
Thanks for bringing this up, English configs look good. Will wait for @rlangman's word on the Spanish values for approval.

File: examples/tts/conf/es/fastpitch_align_44100_ipa.yaml (hunk 1 above)
Comment (XuesongYang) [reply examples/tts/conf/es/fastpitch_align_44100_ipa.yaml]: yeah, talked with @rlangman offline. Revised in the new commit.

**Review by rlangman — DISMISSED** [review]

**Comment by XuesongYang** [comment]
Our CI failed due to lack of pitch_mean: omegaconf.errors.InterpolationToMissingValueError: MissingMandatoryValue while resolving interpolation: Missing mandatory value: pitch_mean
. Details in http://10.110.40.131:8080/blue/rest/organizations/jenkins/pipelines/NeMo/branches/PR-5392/runs/3/nodes/1233/log/?start=0.

@redoctopus is it ok if I override pitch mean and std with LJSpeech's values for our CI tests on an4?

**Comment by redoctopus** [comment]
> Our CI failed due to lack of pitch_mean: `omegaconf.errors.InterpolationToMissingValueError: MissingMandatoryValue while resolving interpolation: Missing mandatory value: pitch_mean` . Details in http://10.110.40.131:8080/blue/rest/organizations/jenkins/pipelines/NeMo/branches/PR-5392/runs/3/nodes/1233/log/?start=0.

Ugh, haha, I think we might be seeing this pop up occasionally for the next while.

> @redoctopus is it ok if I override pitch mean and std with LJSpeech's values for our CI tests on an4?

Sure thing! Not super important for the test that the pitch mean/std is accurate to the toy dataset :P

**Review by redoctopus — APPROVED** [review]
Thanks for updating this!

**Pull request merged**